.shellcheckrc

```diff
@@ -0,0 +1,9 @@
+# Shellcheck configuration file
+
+# SC1090: Can't follow non-constant source. Use a directive to specify location
+# Ignore until https://github.com/koalaman/shellcheck/issues/769 is fixed upstream
+disable=SC1090
+
+# SC2039: In POSIX sh, local is undefined
```

Comment:
comments are full sentences, so put periods at the end of sentences

Reply:
Sure, fixed.